readonly/load_models.py

```diff
@@ -29,6 +29,8 @@
 logging.basicConfig(level="DEBUG")
 log = logging.getLogger(__file__)
 
+WORKSPACE_DIR = os.getenv("WORKSPACE_DIR", "/workspace")
```

Comment:
What are the workspace limitations? The techuser has very limited access permissions. I suppose we only want to support paths below `/workspace`.

The environment variable should also be added to our documentation.